sync_issues_to_jira/sync_issue.py

```diff
@@ -285,17 +302,20 @@ def _add_remote_link(jira, issue, gh_issue):
     Add the JIRA "remote link" field that points to the issue
     """
     gh_url = gh_issue["html_url"]
-    jira.add_remote_link(issue=issue,
-                         destination={"url": gh_url,
-                                      "title": gh_issue["title"],
-                                      },
-                         globalId=gh_url,  # globalId is always the GitHub URL
-                         relationship="synced from")
+    jira.add_remote_link(
+        issue=issue,
+        destination={
+            "url": gh_url,
+            "title": gh_issue["title"],
+        },
+        globalId=gh_url,  # globalId is always the GitHub URL
+        relationship="synced from",
+    )
 
 
 def _update_github_with_jira_key(gh_issue, jira_issue):
-    """ Append the new JIRA issue key to the GitHub issue
-        (updates made by github actions don't trigger new actions)
+    """Append the new JIRA issue key to the GitHub issue
+    (updates made by github actions don't trigger new actions)
     """
     github = Github(os.environ["GITHUB_TOKEN"])
```

Comment:
I would suggest extracting Github object creation to a separate function to avoid duplication